### PR TITLE
Reference Promote_OMR job instead of Promote-OpenJ9-OMR-master-to-openj9

### PR DESCRIPTION
**Pull request by fjeremic**
The former is the new version of the latter, which may get deleted soon.

Signed-off-by: Filip Jeremic <fjeremic@ca.ibm.com>

### CODE REVIEW

**Comment by fjeremic** [comment]
@0xdaryl @AdamBrousseau FYI. @AdamBrousseau we should likely remove the stale job in the future. I didn't want to screw anything up on Jenkins so I'll leave this to you to do if possible.

File: buildenv/jenkins/README.md [hunk 1]
```diff
@@ -224,8 +224,8 @@ Infrastructure pipelines are available [**here**](https://ci.eclipse.org/openj9/
     - Trigger:
         - Build periodically, 15 minutes
 
-- Promote-OpenJ9-OMR-master-to-openj9
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Promote-OpenJ9-OMR-master-to-openj9)](https://ci.eclipse.org/openj9/job/Promote-OpenJ9-OMR-master-to-openj9)
+- Promote_OMR
+    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Promote_OMR)](https://ci.eclipse.org/openj9/job/Promote_OMR)
     - Description:
         - Promotes eclipse/openj9-omr branch master to branch openj9
         - Lays a tag down on the promoted SHA in the format `omr_merge_YYYYMMDD_HHMMSS` with annotations including the current OpenJ9 and OpenJDK SHAs
```
Comment (AdamBrousseau) [inline buildenv/jenkins/README.md]: Should also update tag format which is now `promote_merge_YYYYMMDD_HHMMSS`
https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/jobs/infrastructure/promote_repo.groovy#L68

Comment (fjeremic) [reply buildenv/jenkins/README.md]: Fixed in 220a471

**Comment by AdamBrousseau** [comment]
Jenkins copyright check

**Pull request merged**